CsprSdkStandardTestsNet/Test/Features/info_get_chainspec.feature

```diff
@@ -0,0 +1,8 @@
+Feature: info_get_chainspec
+
+    Scenario: info_get_chainspec
+        Given that the info_get_chainspec is invoked against nctl
```

Comment:
Not sure of the working here as both the nctl commands and sdk are invoked against the nctl node
perhaps:
```
 Given that the info_get_chainspec is invoked against a node using the nctl command line tools
 And that info_get_chainspec is invoked against a node using the SDK API
```